### PR TITLE
docs: update error messages guide with new auto cancellation errors

### LINKED ISSUE #5025: Update the errors reference page to mention these errors that can occur with this new flag
- [errors reference page](https://docs.cypress.io/guides/references/error-messages#CLI-Errors)
- [new errors](https://docs.google.com/spreadsheets/d/1c9wmn6RgplaImt_1B2bXUJvC2IyFPa-OkXLqkeRF-O8/edit#gid=0)
- Add at least references to error # 4, #5 and #6/7 (as one) to this page

### PR DESCRIPTION
**Pull request by elylucas**
closes #5025

### CODE REVIEW

File: docs/guides/references/error-messages.mdx [hunk 1]
```diff
@@ -836,6 +838,32 @@ If you do not have Powershell available, you can also make this change via
...
+
+The `--auto-cancel-after-errors` flag is only available in Cypress 12.x and
+later, and must be used with the `cypress run` command.
+
+### <Icon name="exclamation-triangle" color="red" /> --auto-cancel-after-failures must be a integer or false
+
+You passed in an invalid value for the `--auto-cancel-after-errors` flag. It
+must be a numeric value or false.
+
+### <Icon name="exclamation-triangle" color="red" /> --auto-cancel-after-failures passed without a business or enterprise Cloud account
```
Comment (emilyrohrbough) [inline docs/guides/references/error-messages.mdx:851]:
```suggestion
### <Icon name="exclamation-triangle" color="red" /> --auto-cancel-after-failures passed without a Business or Enterprise Cloud account
```

File: docs/guides/references/error-messages.mdx [hunk 2]
```diff
@@ -836,6 +838,32 @@ If you do not have Powershell available, you can also make this change via
 regedit or gpedit.
 [See Microsoft's documentation for details.](https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation)
 
+### <Icon name="exclamation-triangle" color="red" /> error: unknown option: --auto-cancel-after-failures
+
+The `--auto-cancel-after-errors` flag is only available in Cypress 12.x and
```
Comment (ryanpei) [inline docs/guides/references/error-messages.mdx:843]:
```suggestion
The `--auto-cancel-after-failures` flag is only available in Cypress 12.x and
```

File: docs/guides/references/error-messages.mdx [hunk 3]
```diff
@@ -836,6 +838,32 @@ If you do not have Powershell available, you can also make this change via
...
+later, and must be used with the `cypress run` command.
+
+### <Icon name="exclamation-triangle" color="red" /> --auto-cancel-after-failures must be a integer or false
+
+You passed in an invalid value for the `--auto-cancel-after-errors` flag. It
```
Comment (ryanpei) [inline docs/guides/references/error-messages.mdx:848]:
```suggestion
You passed in an invalid value for the `--auto-cancel-after-failures` flag. It
```

**Comment by ryanpei** [comment]
@elylucas apologies I completely goofed on the error messages for this one. Per the [spreadsheet](https://docs.google.com/spreadsheets/d/1c9wmn6RgplaImt_1B2bXUJvC2IyFPa-OkXLqkeRF-O8/edit#gid=271916282) we are actually now using the messages outlined in column F, not column B as I originally proposed. I can update the text on this one if you don't have availability. It's my bad anyway

**Comment by elylucas** [comment]
> s

no problem i got it

**Comment by elylucas** [comment]
@ryanpei I updated the error messages, but I was a little confused as the text in column F was code. I tried to extract the messages from it, but if these aren't correct let me know.

**Pull request merged**